pkg/promlib/client/transport_test.go

```diff
@@ -24,25 +23,4 @@ func TestCreateTransportOptions(t *testing.T) {
 		require.Equal(t, map[string]string{"foo": "bar"}, opts.Headers)
 		require.Equal(t, 2, len(opts.Middlewares))
 	})
-
-	t.Run("add azure credentials if configured", func(t *testing.T) {
```

Comment:
Since I moved the code in library there shouldn't be any vendor specific code. So this was failing. I will test this case in `pkg/tsdb/prometheus_test.go` file.